Source/VideoView.swift

```diff
@@ -198,6 +196,6 @@ class MovieContainer: UIView {
     }
 
     func updateProgressBar(forDuration duration: Double, currentTime: Double) {
-        self.viewDelegate?.movieContainer(self, didRequestToUpdateProgressBar: duration, currentTime: currentTime)
+        self.viewDelegate?.videoView(self, didRequestToUpdateProgressBar: duration, currentTime: currentTime)
```

Comment:
Much better name. Could be better? Is this the view that  displays the actual video, right? Maybe `videoPreview`?